lib/generator/repository.rb

```diff
@@ -35,6 +35,10 @@ def create_tests_file
     def exercise_name
       @exercise_name ||= slug.tr('-', '_')
     end
+
+    def test_case_name
+      exercise_name + '_case'
```

Comment:
Isn't that true for `#exercise_name` as well? What's the difference between the two?